src/components/include/protocol_handler/session_observer.h

```diff
@@ -43,6 +43,12 @@
          *\namespace protocol_handlerHandler
          *\brief Namespace for SmartDeviceLink ProtocolHandler related functionality.
          */
+struct ExistingSessionInfo {
```

Comment:
@tkireva It would be better if this type is declared as nested of SessionObserver